packages/material-ui/src/Modal/Modal.test.js

```diff
@@ -527,20 +527,6 @@ describe('<Modal />', () => {
     });
   });
 
-  describe('prop: onRendered', () => {
-    it('should fire', () => {
-      const handleRendered = spy();
-
-      render(
-        <Modal open onRendered={handleRendered}>
-          <div />
-        </Modal>,
-      );
-
-      expect(handleRendered).to.have.property('callCount', 1);
-    });
-  });
-
```

Comment:
Deprecated does not mean removed. These test should be kept. We just expect a warning in addition.

Reply:
Removing the test was faster. What's the value in keeping the test? We don't aim to push v4 much further.

Reply:
> Removing the test was faster.

This indicates that we have a knowledge problem that we could fill with better testing documentation/API. What did you try to do that would've been time consuming?

Reply:
> What did you try to do that would've been time consuming?

I made the pull request without running the docs nor the tests locally, it was purely done in the text editor. I imagine any other approach would have required at least some level of trial and error.